examples/criterion-rs/benches/bench.rs

```diff
@@ -0,0 +1,35 @@
+#[macro_use]
+extern crate criterion;
+use criterion::{black_box, Criterion, BenchmarkId};
+use criterion_example::{fib, fast_fib};
+
+fn bench_fib_10(c: &mut Criterion) {
+    c.bench_function("BenchFib10", move |b| {
+        b.iter(|| {
+            let _ = fib(black_box(10));
+        });
+    });
+}
+
+fn bench_fib_20(c: &mut Criterion) {
+    c.bench_function("BenchFib20", move |b| {
```

Comment:
Note that we can add whitespaces and other characters to the bench title that would make the title doesn't match with the extract regex.

Reply:
ok. When someone complains about that, let's consider to support spaces by making output parser smarter.